default.nix

```diff
@@ -261,7 +292,7 @@ in rec {
 
                 shells-ghc = builtins.attrNames (self.predefinedPackages // self.shellPackages);
 
-                shells-ghcjs = [
+                shells-frontend = [
```

Comment:
Ryan pointed out that it's possible to have a ghc frontend so my suggestion of `shells-frontend` isn't quite right. Maybe `shells-web`?